<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vectorWidget</class>
 <widget class="QWidget" name="vectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>331</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>FontAwesome</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>FreeSans</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: grey</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="xContainer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSlider" name="xValueEdit">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="xValueText">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>FontAwesome</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>0.0</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="yLabel">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>FreeSans</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: grey</string>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="yContainer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSlider" name="yValueEdit">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="yValueText">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>FontAwesome</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="zLabel">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>FreeSans</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: grey</string>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="zContainer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSlider" name="zValueEdit">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="zValueText">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>FontAwesome</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="wLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>FreeSans</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: grey</string>
        </property>
        <property name="text">
         <string>W</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QWidget" name="wContainer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSlider" name="wValueEdit">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="wValueText">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>FontAwesome</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>xValueEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vectorWidget</receiver>
   <slot>valueUpdated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yValueEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vectorWidget</receiver>
   <slot>valueUpdated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zValueEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vectorWidget</receiver>
   <slot>valueUpdated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wValueEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vectorWidget</receiver>
   <slot>valueUpdated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valueUpdated()</slot>
 </slots>
</ui>
